cases/src/narrowing_integer_right_shift_cases.rs

```diff
@@ -53,7 +53,7 @@ pub fn test_narrowing_integer_right_shift() {
         });
     }
     let sew = 256u64;
-    for lmul in [-2, 1, 4] {
+    for lmul in [-2, 1, 2] {
```

Comment:
why is the range is changed?

Reply:
Sorry, The version of ckb-vm I used for debugging is a bit low, and it can be updated after.